### PR TITLE
Update graphiql from 0.8.1 to 0.10.2

### PR DESCRIPTION
**Pull request by tiegz**
JS package updates:

* graphiql [0.8.1 ~> 0.10.2](https://github.com/graphql/graphiql/compare/v0.8.1...v0.10.2)
* react [15.4.1 ~> 15.5.4](https://github.com/facebook/react/compare/v15.4.1...v15.5.4)

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -39,6 +39,7 @@ task :update_graphiql do
 
   FileUtils.mkdir_p(update_path)
   FileUtils.cd(update_path) do
+    sh("npm init --force")
```
Comment (tiegz) [inline Rakefile]: the `graphiql_update` folder needs a `package.json` file to install the npm packages

File: graphiql-rails.gemspec [hunk 2]
```diff
@@ -1,6 +1,7 @@
 $LOAD_PATH.push File.expand_path("../lib", __FILE__)
 
 require "graphiql/rails/version"
+require "date"
```
Comment (tiegz) [inline graphiql-rails.gemspec]: ruby 2.5 gemifies `date`, so bundler stopped requiring it by default

**Comment by rmosolgo** [comment]
Awesome, thanks! I'll fix the CI fails on master.

**Comment by rmosolgo** [comment]
Shipped as `1.4.2`! I got an error from Rubygems during the push, but it looks like the new gem version _is_ there ...

**Comment by tiegz** [comment]
🙇

**Pull request merged**